frontend/components/SetupPage/Create/SetupBridgeOnboarding/BridgeInProgress.tsx

```diff
@@ -87,14 +83,14 @@ const useBridgingSteps = (quoteId: string) => {
           })();
 
           return {
-            symbol: symbols[index],
+            symbol: tokenSymbols[index],
             status,
             txnLink: step.explorer_link,
           };
         }),
       };
     },
-    // refetch every 5 seconds until the status is FINISHED
+    // fetch every 5 seconds until the status is FINISHED
```

Comment:
I'd rather do this so in case the value is changed we don't need to update the comment
```suggestion
    // fetch by interval until the status is FINISHED
```